src/index.ts

```diff
@@ -82,3 +85,28 @@ export const parseKind = (kind: string, { rootSeparator, groupSeparator }: Separ
     groups,
   };
 };
+
+/**
+ * csf story metadata attached to the story export function
+ */
+export interface StoryMetadata {
+  story?: {
+    /**
+     * story name if differnet from the export name
+     */
+    name?: string;
+
+    /**
+     * optional collection of properties, which values
+     * will be passed onto the story function
+     */
+    properties?: StoryProperties;
```

Comment:
I'm not sure about the naming here "properties" -- other options that have been suggested include: "args", "values", "controls". Properties is quite long and perhaps easily confused with React "props" which they are very similar to but not the same.

Also, have we decided this is a top-level field on the story metadata already?

Reply:
i can actually go for props - since we already named the enum PropertyTypes so i think we should go consistent on this - properties or props
For the top level, i thought we had already decided :)

Reply:
I like the word `controls` a lot too. I was also pondering `data` or `dataTypes`.

Reply:
controls are rather called the property editors in the UI. The definitions here i cant see being called controls.